Exercise_A/index.html

```diff
@@ -6,33 +6,217 @@
+          There are hundreds of templates to choose from, but go ahead and pick this same exact template from the first result on google,
+          edit a few lines, and you're on your way! No one will notice!</p>
+     </div>
+     <div class="d-flex justify-content-center align-content-center">
+      <a href="http://lmgtfy.com/?q=bootstrap+template" target="_blank" class="btn btn-lg" style="background-color: white" href="#" role="button" id="learnMore">GOOGLE THAT SHIT</a>
+    </div>
+    </div>
+
+
+
+    <!--Four icons div-->
+    <div class="container-fluid" id="fourIconsDiv">
+
+          <div id="fourIcons" class="d-flex justify-content-center align-content-center">
+            <h3>So Fucking Creative</h3>
+          </div>
+      </div>
+      <hr id="orangeHR">
+      <div class="row">
+      
+            <div class="col-xs-12 col-sm-12 col-md-6 col-xl-3" id="iconFlex">
+              <div class="secondBox">
+              <i class="fas fa-globe-americas fa-3x" style="color: #f05f40"></i>
+              <h3>This Fucking Template</h3>
+              <p>Guaranteed to use the same fucking template that every other bootstrap website uses, downloaded straight from The Web™</p>
+            </div>
+            </div>
+      
+            <div class="col-xs-12 col-sm-12 col-md-6 col-xl-3" id="iconFlex">
+              <div class="secondBox">
+              <i class="fas fa-paper-plane fa-3x" style="color: #f05f40"></i>
+              <h3>This Set of Four Icons</h3>
+              <p>Look at this cool set of four icons describing different things about us! We use four, because it's the default.</p>
+            </div>
+          </div>
+      
+            <div class="col-xs-12 col-sm-12 col-md-6 col-xl-3" id="iconFlex">
+              <div class="secondBox">
+              <i class="fas fa-thumbs-up fa-3x" style="color: #f05f40"></i>
+              <h3>Lots of effort</h3>
+              <p>We even changed some of the icons! We take the extra effort to make our designs truly original.</p>
+            </div>
+          </div>
+      
+            <div class="col-xs-12 col-sm-12 col-md-6 col-xl-3" id="iconFlex">
+              <div class="secondBox">
+              <i class="fas fa-heart fa-3x" style="color: #f05f40"></i>
+              <h3>Made with Love</h3>
+              <p>Because nothing says hard work and talent like editing a few lines of text.</p>
+            </div>
+          </div>
+      
+          </div>
+      
+        </div>
+
+
+
+
+
+
+
+        <!-- Four pictures with overlay etc -->
+  <div class="container-fluid" id="fourIconsDiv">
+    <div class="row">
+
+      <div class="col-xs-12 col-sm-12 col-md-6 col-xl-3" id="fourIconsDiv">
+      <div class="adjustedPicture">
+        <img src="./assets/images/1.jpg">
+        <div class="overlay">
+          <div class="text"><span style="color: rgb(255, 251, 251, 0.8);">ONE THING </span><br> Look</div>
+        </div> 
+    </div>
+    </div>
+

```

Comment:
Great implementation of these pictures and overlays. 
Superb use of the column system to define how many columns you want per break point. 
Try to use bootstrap utility classes instead of inline style.
Or style in your style sheets.